mono/mini/main.c

```diff
@@ -159,7 +159,9 @@ bundle_save_library_initialize (void)
 	g_free (path);
 	if (bundled_dylibrary_directory == NULL)
 		return;
+#ifdef HAVE_ATEXIT
```

Comment:
Atexit is ANSI C89 and should be as portable as malloc, sprintf, exit, strlen, etc.